Automattic/wp-calypso
# Progress bar: Rename 'Activating theme' step

Fixes https://github.com/Automattic/wp-calypso/issues/83205

## Proposed Changes

Rename `"Activating theme"` step to `"Getting the theme ready"` as the theme is not activated at this step anymore.

![CleanShot 2023-10-18 at 15 44 02@2x](https://github.com/Automattic/wp-calypso/assets/5039531/dbd21f25-3687-4e98-b567-85d50dbe4c75)


## Testing Instructions
* With a non-atomic site
* Go to the Design Picker page with a Marketplace theme selected /setup/update-design/designSetup?siteSlug=:site&theme=:theme_slug. Theme slugs such as makoney and olsen-fse can be used.
* Click on Unlock Theme
* Complete the purchase
* Check the ` 'Activating theme'` step is not shown anymore

## Pre-merge Checklist
